pysindy/__init__.py

```diff
@@ -4,9 +4,9 @@
 try:
     __version__ = get_distribution(__name__).version
 except DistributionNotFound:
+    __version__ = "v1.4.3"  # required for sphinx-build to compile locally
```

Comment:
I don't know that this should go into the master branch.

Reply:
agreed, originally added this to build sphinx successfully on my local computer, but deleting this line now